RestaurantReservation.Db/Seeders/CustomerSeeder.cs

```diff
@@ -0,0 +1,23 @@
+﻿using RestaurantReservation.Db.Models;
+
+namespace RestaurantReservation.Db.Seeders;
+
+public class CustomerSeeder : IDataSeeder<Customer>
```

Comment:
seeders shouldn't be public and exposed, they are only consumed from within the DB layer.